contrib/time_series_expectations/docs/README.md

```diff
@@ -0,0 +1,176 @@
+    context.sources.my_db.assets.my_table,
+    columns=["foo", "bar", "baz"],
+    metric="percent_matching_regex",
+    kwargs={
+        "regex": "\d+",
+    }
+)
+```
+
+
+* RowLevelTimeSeries: "I have numeric time series data in a table, file, etc. I want to learn what patterns are normal in that data, and then monitor it for anomalies"
+
+time_series_expectation_suite = context.assistants.time_series_by_row.profile_data_asset(
+    context.sources.my_db.assets.my_table,
+    date_column="created_at",
+    columns=["foo", "bar", "baz"],
+    metrics=["mean", "median", "sum", "percent_null"]
+)
+
+### New Expectations
+
+This module introduces several new Expectations, inheriting from some new abstract base classes.
+
+You can learn more about the Expectations in the Expectation gallery, [here](link).
+
+
+### Abstract Base Classes
+
+The most important ABCs are [BatchAggregateStatisticTimeSeriesExpectation](link), [ColumnAggregateTimeSeriesExpectation](link), and  [ColumnPairTimeSeriesExpectation](link). They allow time series models to be applied to data in a variety of shapes and formats. Please see the class docstrings for more detailed explanation.
+
+The full class hiereachy is:
+
+    *TableExpectation* (ABC)
+        BatchAggregateStatisticExpectation (ABC)
+            ExpectBatchAggregateStatisticToBeBetween (ABC)
+                expect_batch_update_time_to_be_between
+                expect_batch_volume_to_be_between
+
+            BatchAggregateStatisticTimeSeriesExpectation (ABC)
+                ExpectBatchAggregateStatisticToMatchProphetDateModel (ABC)
+                    expect_batch_volume_to_match_prophet_date_model
+
+                ExpectBatchAggregateStatisticToMatchProphetTimestampModel (ABC)
+                    expect_batch_volume_to_match_prophet_timestamp_model
+
+                ExpectBatchAggregateStatisticToMatchArimaModel (ABC)
+                    expect_batch_volume_to_match_arima_model
+
+        *ColumnExpectation* (ABC)
+            ColumnAggregateTimeSeriesExpectation (ABC)
+                expect_column_{property}_to_match_{model}_model
+
+                    properties:
+                        max
+                        min
+                        stdev
+                        sum
+                        percent_missingness
+                        percent_matching_regex
+                        ...
+
+                    models:
+                        prophet_date
+                        prophet_timestamp
+                        arima
+
+        *ColumnPairMapExpectation* (ABC)
+            ColumnPairTimeSeriesExpectation (ABC)
+                expect_column_pair_values_to_match_prophet_date_model
+                expect_column_pair_values_to_match_prophet_timestamp_model
+                expect_column_pair_values_to_match_arima_model
+        
+Formatting conventions:
+
+* Abstract base classes are in camel case, and marked with (ABC).
+* Concrete classes are in snake case.
```

Comment:
I think this might be confusing to have class names in snake case. Could we continue with our convention of using the snake name for use with the `validator` and in `ExpectationConfiguration` and use the camel case in expectation definition?

Reply:
As discussed, I've moved this text from `docs/README.md` to the main `README.md`, and clarified that the snake vs camel case distinction is only within the README, not the actual code.